content/token-program.md

```diff
@@ -65,6 +65,8 @@ Under the hood, the `createMint` function is simply creating a transaction that
 1. Create a new account 
 2. Initialize a new Mint 
 
+JC nit: Maybe explain why this is important -- that if you do them in separate transactions, someone else could scoop your created account for their own mint!
```

Comment:
@joncinque Does adding this on line 104 after the code snippet fit what you're suggesting?

> When manually building the instructions to create a new token mint, make sure you add the instructions to create the account and initialize the mint to the same transaction. If you were to do each step in a separate transaction, it's theoretically possible for somebody else to take the account you create and initialize it for their own mint.